index.js

```diff
@@ -61,7 +61,7 @@ app.get(basePath + '*', function(req, res) {
       res.render('index.html', {
         base: basePath,
         prefetchedItem: JSON.stringify(body),
-        title: body.page_title,
```

Comment:
what's this change for?

Reply:
Ho yeh, i forgot to ask you. I saw in all the office tenders that the title is 'None עם None'. I didnt find in the project the code which generate it, so i assume it came from the server, and i solved it for now in this ugly way